packages/internal/package.json

```diff
@@ -22,6 +22,7 @@
   "dependencies": {
     "@agoric/assert": "^0.6.0",
     "@agoric/base-zone": "^0.1.0",
+    "@agoric/whenable": "^0.1.0",
```

Comment:
yet another reason to consider moving internal/whenable to vat-data.

Reply:
> consider moving internal/whenable to vat-data.

Hmm.  That would work, I guess.

Reply:
If this turns out to be easy, please do so in this PR before merge, because it'll be more painful afterwards. If you run into unexpected difficulties, then we can postpone to a follow-on PR. But please try.